XRTK.WindowsMixedReality/Packages/com.xrtk.wmr/Runtime/Extensions/InteractionSourceExtensions.cs

```diff
@@ -42,7 +42,13 @@ public static void StartHaptics(this InteractionSource interactionSource, float
 
         public static void StartHaptics(this InteractionSource interactionSource, float intensity, float durationInSeconds)
         {
-            if (!WindowsApiChecker.UniversalApiContractV4_IsAvailable && !Application.isEditor)
+            // GetForCurrentView and GetDetectedSourcesAtTimestamp were both introduced in the same Windows version.
+            // We need only check for one of them.
+            if ((!WindowsApiChecker.IsMethodAvailable(
```

Comment:
Let's change this later when we go full native

Reply:
Why keep waiting for things when they can be done right now?

Reply:
I agree, if this update works for the current release, sure update when we go native if needed.  But that shouldn't stop a fix going in now if it works.

Reply:
Because I wanna go full native before the release. So we will effectively be doing it right after the hands and other things make it in. Just wanna understand what the root cause of the problem is before we make changes